public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -164,6 +164,11 @@ export class ElasticDatasource {
           }
         }
 
+        var timeAsNumber = _.toNumber(time);
+        if (!_.isNaN(timeAsNumber)) {
```

Comment:
should this not be the other way around, if time is string then try to parse it to number?

Reply:
well elastic can send time as a string (2017.01.01 etcc)...